### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lukacsja**
[DEMO LINK](https://lukacsja.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/UserInfo/UserInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  </>
 );
```
Comment (Mohameddahi) [inline src/components/UserInfo/UserInfo.tsx:14]: It is better to use destructing here.

**Review by Mohameddahi — APPROVED** [review]
in general you did great, just consider that comment.

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': completed,
+      })}
```
Comment (meforoma) [inline src/components/TodoInfo/TodoInfo.tsx:21]:
```suggestion
      className={classNames(
        'TodoInfo',
        { 'TodoInfo--completed': completed },
      )}
```

File: src/components/TodoInfo/TodoInfo.tsx [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      {user
+        && <UserInfo user={user} />}
```
Comment (meforoma) [inline src/components/TodoInfo/TodoInfo.tsx:28]:
```suggestion
      {user && <UserInfo user={user} />}
```

File: src/components/UserInfo/UserInfo.tsx [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  </>
```
Comment (meforoma) [inline src/components/UserInfo/UserInfo.tsx:13]: do you actually need a fragment here?
```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
```

**Review by meforoma — APPROVED** [review]
few minor editorials, otherwise - ✅ 👍🏼

**Pull request closed without merging**